packages/nimble-components/src/anchor-menu-item/specs/README.md

```diff
@@ -61,16 +61,10 @@ _The key elements of the component's public API surface:_
 
 ```html
 <template role="menuitem" aria-disabled="${x => x.disabled}">
-    <a
-        download="${x => x.download}"
-        href=${x => (x.disabled ? null : x.href)}
-        hreflang="${x => x.hreflang}"
-        ping="${x => x.ping}"
-        referrerpolicy="${x => x.referrerpolicy}"
-        rel="${x => x.rel}"
-        target="${x => x.target}"
-        type="${x => x.type}"
-    >
+    <a download="${x => x.download}" href="${x" ="">
```

Comment:
This is harder to read and probably no longer syntactically correct. Can we revert it? I'm guessing it came from prettier?

Reply:
It came from prettier but it's also spec text and not production code so don't feel like it's worth the effort. Probably an example of too much implementation detail in spec anyway.